library/objective-c/EnvoyEngine.mm

```diff
@@ -23,45 +164,92 @@ + (EnvoyStatus)runWithConfig:(NSString *)config logLevel:(NSString *)logLevel {
   }
 }
 
-+ (EnvoyStream)startStreamWithObserver:(EnvoyObserver *)observer {
-  NSLog(@"%@ not implemented, returning failure", NSStringFromSelector((SEL) __func__));
-  EnvoyStream stream;
-  stream.status = EnvoyStatusFailure;
-  return stream;
++ (void)setupEnvoy {
+  setup_envoy();
+}
+
+// TODO: move to Envoy
++ (EnvoyStream *)startHttpStreamForRequest:(EnvoyRequest *)request handler:(EnvoyObserver *)handler {
```

Comment:
Let's go ahead and update this in the header file

Reply:
Working on getting that updated.